apps/fz_http/lib/fz_http/authorization.ex

```diff
@@ -0,0 +1,11 @@
+defmodule FzHttp.Authorization do
+  # def get_subject(%FzHttp.Users.User{} = user, user_ip, user_agent) do
```

Comment:
Just double-checking that leaving these commented out was intentional

Reply:
I'll remove it, it will be part of the next PR.